lncfg/address.go

```diff
@@ -162,6 +165,59 @@ func ParseAddressString(strAddress string, defaultPort string,
 	}
 }
 
+// ParseLNAddressString converts a string of the form <pubkey>@<addr> into an
+// lnwire.NetAddress. The <pubkey> must be presented in hex, and result in a
+// 33-byte, compressed public key that lies on the secp256k1 curve. The <addr>
+// may be any address supported by ParseAddressString. If no port is specified,
+// the defaultPort will be used. Any tcp addresses that need resolving will be
+// resolved using the custom tcpResolver.
+func ParseLNAddressString(strAddress string, defaultPort string,
```

Comment:
Nice! We can eventually also use this method in the `ConnectPeer` RPC method, and also possibly in a few areas in `lncli`.